Classes/NewComment/NewCommentViewController.swift

```diff
@@ -68,6 +69,11 @@ final class NewCommentViewController: UIViewController, UITextViewDelegate, AddC
         textView.delegate = self
         textView.font = Styles.Fonts.body
         textView.backgroundColor = .clear
+        textView.inputAccessoryView = formattingController.view
+        formattingController.textView = textView
+        // need to constraint here, maybe better idea swhere else
+        // Otherwise takes all screen
+        textView.inputAccessoryView?.frame = CGRect(origin: .zero, size: CGSize(width: 0.0, height: 44.0))
```

Comment:
I would prefer if that was handled but the controller, but I don't seem to get the constraint working :/